.github/workflows/ci.yml

```diff
@@ -64,21 +62,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [18.x, 20.x, 22.4.1]
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
+1 to including v22. @kriskowal since Node 18 EOLed a couple days ago, are you okay with removing it?

Reply:
That'd be a breaking change...

Reply:
FWIW, if we drop Node.js v18, then all of LavaMoat is stuck until we can drop v18.

Reply:
When can LavaMoat drop v18?

Reply:
I endo can continue to support Node 18 for longer than EOL. There probably should be a policy about this that endo maintainers can reference.

Reply:
@turadg Not anytime in the near future

Reply:
@mhofman @turadg I've created an agenda item for the next meeting to discuss such a policy.